lib/solidus_braintree.rb

```diff
@@ -8,6 +8,7 @@
 require 'solidus_braintree/extension_configuration'
 require 'solidus_braintree/version'
 require 'solidus_braintree/engine'
+require 'solidus_paypal_braintree'
```

Comment:
Have we considered not including this file by default? The idea is not to have the alias for users that don't need it. Would it make the upgrade path more difficult?

Reply:
@waiting-for-dev I'll test if the user can manually add it after installing the extension's version update.